web-console/knikubevirt/clone-vm/clone-vm.md

```diff
@@ -0,0 +1,43 @@
+# Clone VM
```

Comment:
Are we ok with using "VM" in place of virtual machine? @lizsurette have you thought about this at all?

Reply:
@matthewcarleton - I went back and forth when I created the initial PR with a starter list. Since it's repeated a ton I went with the shorter option, but maybe others can comment who would likely be the audience of this page in reviewing/referring back to designs... @beanh66 @cshinn @bmignano @alimobrem - This is referring to what we call the link in the main homepage to these features as well as the header title.